rules/Php80/Rector/Class_/StringableForToStringRector.php

```diff
@@ -106,6 +106,12 @@ public function refactor(Node $node): ?Node
         $classLikeAncestorNames = $this->familyRelationsAnalyzer->getClassLikeAncestorNames($node);
 
         if (in_array(self::STRINGABLE, $classLikeAncestorNames, true)) {
+            // add return type
+            if ($toStringClassMethod->returnType === null) {
+                $toStringClassMethod->returnType = new Identifier('string');
+                return $node;
+            }
```

Comment:
when parent interface already extends Stringable, it add return type: `string`, without `re-adding` Stringable into implements 👍

see https://3v4l.org/vgKoK